vtex-sites/storecomponents.store
# Chore | Search and Home view

## What's the purpose of this pull request?
Move files to use the new view filesystem API. More info at:
https://github.com/vtex/faststore/pull/739

## How to test it?
Open the home and search pages. Note that nothing has changed from the production version of the site
